### PR TITLE
Update video input controls, rename device permission status and device selection methods

### PR DESCRIPTION
**Pull request by xuesichao**
**Issue #:** 

**Description of changes:**
- Rename device selection methods in MeetingManager and remove hooks that expose them.
- Video input controls do not turn on camera when switching devices
- Rename device permission stauts to device label trigger status

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Install in meeting demo and confirm the demo works well.

3. If you made changes to the component library, have you provided corresponding documentation changes?
Yes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,11 +16,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Removed `useSelectAudioInputDevice`, `useSelectAudioOutputDevice` and `useSelectVideoInputDevice` hook.
 
 ### Changed
 
 - Revert "Add Observer to select input device error" ([PR #493](https://github.com/aws/amazon-chime-sdk-component-library-react/pull/493)). `useAudioInputs` and `useVideoInputs` hook no longer return `selectDeviceError`. `selectAudioInputDevice`, `selectVideoInputDevice`, and `selectAudioOutputDevice` method of `MeetingManager` now throw error when failed. The device selection methods returned by `useSelectAudioInputDevice`, `useSelectVideoInputDevice`, and `useSelectAudioOutputDevice` hook are built on top of these `MeetingManager` methods, thus now they throw error when failed as well.
 - Change the `selectedDeviceId: string | null` to `selectedDevice: Device | TransformDevice`.
+- Rename `selectAudioInputDevice` to `startAudioInputDevice`, `selectVideoInputDevice` to `startVideoInputDevice`, and `selectAudioOutputDevice` to `startAudioOutputDevice`.
+- Rename `DevicePermissionStatus.UNSET` to `DevicePermissionStatus.UNTRIGGERED` and `DevicePermissionStatus` to `DeviceLabelTriggerStatus`.
```
Comment (devalevenkatesh) [inline CHANGELOG.md:26]: nit: UNTRIGGERED -> NOT_TRIGERRED ?

File: src/hooks/sdk/docs/useDeviceLabelTriggerStatus.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<Meta title="SDK Hooks/useDevicePermissionStatus" />
+
+# useDeviceLabelTriggerStatus
+
+The `useDeviceLabelTriggerStatus` hook returns the `DeviceLabelTrigger` status from the enum below.
+
+It tracks if application has triggered the `DeviceLabelTrigger` to request device permission from browser, and if the request is granted or denied.
```
Comment (ltrung) [inline src/hooks/sdk/docs/useDeviceLabelTriggerStatus.mdx:7]: I think we want to explain what each of these means in more details, esp `UNTRIGGERED` which can means either it is not triggered yet or the labels are already there so no need to trigger.

File: src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx [hunk 3]
```diff
@@ -89,19 +89,21 @@ const VideoInputBackgroundBlurControl: React.FC<Props> = ({
...
             logger.error('Transform device cannot choose new inner device');
           }
-          setIsLoading(false);
+        }
+
+        if (isVideoEnabled) {
+          await meetingManager.startVideoInputDevice(newDevice);
         } else {
-          await selectVideoInput(deviceId);
+          meetingManager.selectVideoInputDevice(newDevice);
```
Comment (devalevenkatesh) [inline src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx:106]: I thought we will be maintaining state locally in this component or we keep it now in meetingManager and publish an update just on selection?

Comment (devalevenkatesh) [reply src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx:106]: I got this, we initially decided to go with local UI component state but it seems moved to meetingManager now.

Comment (xuesichao) [reply src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx:106]: Yes, discussed with Trung offline, moved to `meetingManager`. Forgot telling you. :p

File: src/providers/MeetingProvider/MeetingManager.ts [hunk 4]
```diff
@@ -500,9 +505,14 @@ export class MeetingManager implements AudioVideoObserver {
     }
   };
 
+  selectVideoInputDevice = (device: VideoInputDevice): void => {
```
[the remark this answers is not in the document]
Comment (xuesichao) [reply src/providers/MeetingProvider/MeetingManager.ts:508]: We don't have similar use cases for audio devices. The main difference is requiring video media stream will make camera LED ON. We can always add such APIs later.

File: src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx [hunk 5]
```diff
@@ -89,19 +89,21 @@ const VideoInputBackgroundBlurControl: React.FC<Props> = ({
       try {
         // If background blur is on, then re-use the same video transform pipeline, but replace the inner device
         // If background blur is not on, then do a normal video selection
+        let newDevice: VideoInputDevice = deviceId;
         if (isVideoTransformDevice(selectedDevice) && !isLoading) {
-          setIsLoading(true);
+          setIsLoading(false);
```
[the remark this answers is not in the document]
Comment (xuesichao) [reply src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx:94]: Oops, this is typo, should be `true`

**Review by devalevenkatesh — APPROVED** [review]

File: src/hooks/sdk/docs/useDeviceLabelTriggerStatus.mdx (hunk 2 above)
Comment (xuesichao) [reply src/hooks/sdk/docs/useDeviceLabelTriggerStatus.mdx:7]: >esp UNTRIGGERED which can means either it is not triggered yet or the labels are already there so no need to trigger.

Yeah, this is why I rename it from `unset` to `untriggered`. `unset` is very confusing and `untriggered` is the actual status.

**Pull request merged**